zerver/tests/test_user_groups.py

```diff
@@ -110,6 +110,14 @@ def test_user_group_create(self) -> None:
         self.assert_json_error(result, "User group 'support' already exists.")
         self.assert_length(UserGroup.objects.all(), 2)
 
+    def test_user_group_get(self) -> None:
+        # Test success
+        user_profile = self.example_user('hamlet')
+        self.login(self.example_email("hamlet"))
```

Comment:
You can grab the email here with `user_profile.email`.

Reply:
OK, have changed that.